### ISSUE TITLE
Can't change themes when I have a custom-test-theme.css file in maloja/custom_css

### ISSUE THREAD

**Issue by puelp** [krateng/maloja#280]
I typed in the name of a default theme in the web interface setting, but the theme didn't change. I checked the settings .ini and the theme had been inputted and saved correctly. Still, the theme refused to change.

So I stopped the server, went into the custom_css folder and moved my custom-test-theme.css file elsewhere, and the theme changing started to work again.

**Comment by duckfromdiscord** [comment]
I didn't know we had themes!

**Comment by puelp** [comment]
Yeah, I only found out how to see it after seeing it in the web admin settings and then through trial and error that took me quite a bit of time.

I opened some new issues related to this:

https://github.com/krateng/maloja/issues/285
https://github.com/krateng/maloja/issues/286
